lib/editor/components/ExceptionDate.js

```diff
@@ -104,68 +102,64 @@ export default class ExceptionDate extends Component<Props, State> {
       dateTimeProps.defaultText = 'Select date'
     }
     return (
-      <div
-        key={index}
-        style={{
-          alignItems: 'center',
-          display: 'grid',
-          gap: 5,
-          gridTemplateColumns: 'repeat(12, 1fr)',
-          justifyContent: 'center',
-          textAlign: 'center',
-          position: 'relative',
-          marginBottom: '5px'
-        }}>
+      <FormGroup validationState={validationState}>
         <div
```

Comment:
Do we need this `<div>` now that `<FormGroup>` wraps everything? (We still need the props passed to that `<div>`)

Reply:
Nope, good call. Removed and applied formatting to `<FormGroup>` in https://github.com/ibi-group/datatools-ui/pull/828/commits/bd59f94bd41bd062d40630c054cacfdc97f32a98